lib/compat/wordpress-6.2/block-editor-settings.php

```diff
@@ -13,6 +13,8 @@
  * @return array New block editor settings.
  */
 function gutenberg_get_block_editor_settings_6_2( $settings ) {
+	$can_user_edit_global_custom_css = apply_filters( 'enable_global_styles_custom_css', current_user_can( 'unfiltered_html' ) );
```

Comment:
```suggestion
	$can_user_edit_global_custom_css = apply_filters( 'enable_global_styles_custom_css', current_user_can( 'edit_css' ) );
```

I think this should check the `edit_css` capability by default, so it's consistent with the Custom CSS feature in the Customizer: https://core.trac.wordpress.org/browser/trunk/src/wp-includes/customize/class-wp-customize-custom-css-setting.php?rev=51785#L43